single-profile.php

```diff
@@ -17,16 +17,16 @@
 	?>
 
 	<?php
-	$role            = get_the_terms( get_the_ID(), 'role' );
+	$roles           = get_the_terms( get_the_ID(), 'role' );
 	$default_heading = get_theme_mod( 'wmf_related_profiles_heading', __( 'Other members of ', 'shiro-admin' ) );
 	$team_name       = '';
-	$parent_name     = $role[0]->name;
-	$parent_link     = get_term_link( $role[0] );
-    $connected_user = get_post_meta( get_the_ID(), 'connected_user', true );
+	$parent_name     = $roles[0]->name;
+	$parent_link     = get_term_link( $roles[0] );
+	$connected_user = get_post_meta( get_the_ID(), 'connected_user', true );
 
-	if ( ! empty( $role ) && ! is_wp_error( $role ) ) {
-		$team_name = $role[0]->name;
-		$ancestors = get_ancestors( $role[0]->term_id, 'role' );
+	if ( ! empty( $roles ) && ! is_wp_error( $roles ) ) {
+		$team_name = $roles;
```

Comment:
~This variable assignment doesn't seem necessary. Are we using it anywhere?~

Never mind, I see it being passed into the `header/profile-single` template part. 

As a purist I'm bothered by the variable name being single when it's expected to be passed as an array, but that ultimately doesn't matter at all :)